src/main/java/lotto/domain/LottoStatistics.java

```diff
@@ -0,0 +1,24 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class LottoStatistics {
+
+    private LottoStatistics() {
+        throw new UnsupportedOperationException("해당 클래스는 인스턴스를 생성할 수 없습니다.");
+    }
+
+    public static LottoWinningResults winningStatistics(
+            final List<List<Integer>> lottoResults,
+            final List<Integer> lastWeekWinningNumbers
+    ) {
+        LottoWinningResults lottoWinningResults = new LottoWinningResults();
+        lottoResults.stream().forEach(lottoResult -> {
```

Comment:
`forEach`는 `stream`을 생성하지 않아도 사용 할 수 있습니다 🙇 
```suggestion
        lottoResults.forEach(lottoResult -> {
```